.travis.yml

```diff
@@ -58,20 +58,12 @@ matrix:
          # Order of tests run in build stages
          - python: 3.6
            stage: Initial tests
-           env: SETUP_CMD='test --coverage'
```

Comment:
So what does this run now?

Reply:
the same thing, it just uses the default value

Reply:
Oh that was the default value already. Nevermind.